Structure_Engine/Query/TorsionalConstant.cs

```diff
@@ -107,20 +107,20 @@ public static double TorsionalConstant(this FabricatedBoxProfile profile)
 
         /***************************************************/
 
-        [Description("Calcualtes the Torsinal constant for the profile. Note that this is not the polar moment of inertia.")]
+        [Description("Calcualtes the Torsinal constant for the profile. Note that this is not the polar moment of inertia. Formulation taken from BS EN 10210-2:2019.")]
```

Comment:
Calculates the torsional constant for the profile in accordance with BS EN 10210-2:2019. Note that this is not the polar moment of intertia.

Reply:
Done